app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -315,8 +315,14 @@ export default {
     },
     lastHeader: {
       immediate: true,
-      handler() {
-        if (this.session.signedIn) {
+      handler(newHeader) {
+        const waitTwentyBlocks = Number(newHeader.height) % 20 === 0
+        if (
+          this.session.signedIn &&
+          waitTwentyBlocks &&
+          this.$route.name === `validator` &&
+          Number(this.myBond) > 0
```

Comment:
what happens if you load this page first and the bond has not been loaded yet?

Reply:
I haven't run into that issue

Reply:
Tested it on: https://22799-99653950-gh.circle-artifacts.com/0/tmp/voyager/app/dist/index.html?insecure=true#/staking/validators/cosmosvaloper1pg6d6h60y5a32cs9vegd98mlwtlyrf7zxav2ef
- create account
- delegate
- go on validator page
- refresh